src/cap-description.adoc

```diff
@@ -124,6 +138,37 @@ If a capability grants <<r_perm>> and <<c_perm>>, but no <<lm_perm>>, then a cap
 The rules specified by <<ACPERM>> are followed when <<w_perm>> and <<lm_perm>> are removed, so additional permissions may also be removed.
 Clearing a capability's <<lm_perm>> and <<w_perm>> allows sharing a read-only version of a data structure (e.g. a tree or linked list) without making a copy.
 
+[#sl_perm,reftext="SL-permission"]
+Store Level Permission (SL):: This permission allows limiting the propagation of capabilities using the following logic: capabilities with a <<section_cap_level>> less than the inverse of <<sl_perm>> will be stored with the tag cleared.
+With one bit for SL and CL each, this permission functions as follows:
+// This awkward wording above allows a multi-bit scheme:
```

Comment:
Would it help to describe the permission as only allowing writes of other capabilities whose level is greater or equal than our own? (Or at least that how I understood it!)

Reply:
It allows storing capabilities with `CLl >= ~SL`. I added a new table for multi-bit. Happy to change this wording to be clearer, but can't think of anything easy to parse.